chain/index/events.go

```diff
@@ -280,6 +282,39 @@ func (si *SqliteIndexer) checkTipsetIndexedStatus(ctx context.Context, f *EventF
 	return ErrNotFound // Tipset is not indexed
 }
 
+func (si *SqliteIndexer) checkRangeIndexedStatus(ctx context.Context, f *EventFilter) error {
+	startCid, err := si.getTipsetKeyCidByHeight(ctx, f.MinHeight)
+	if err != nil {
+		if errors.Is(err, ErrNotFound) {
+			// Null round for the start of the range is acceptable
+			return nil
+		}
+		return xerrors.Errorf("failed to get tipset key cid for start height: %w", err)
+	}
```

Comment:
In the `ErrNotFound` case, I'd like it to increment by `1` until it finds one <= `MaxHeight` that's not a null round.

In the special case where there are only null rounds between `MinHeight` and `MaxHeight`, pretend that it's indexed and don't return an error, they'll just get no results.

So you'll need a loop here, and a local copy of `f.MinHeight` to increment.